Source/AtomicApp/Player/IPCPlayerApp.cpp

```diff
@@ -206,6 +213,12 @@ namespace Atomic
 
     void IPCPlayerApp::Stop()
     {
+        if (debugPlayer_)
+        {
+            context_->GetSubsystem<JSDebugger>()->AutoReconnect = false;
+            context_->GetSubsystem<JSDebugger>()->Shutdown();
```

Comment:
The problem here is that once the socket disconnects, it starts listening for a connection again, which blocks the destructor from firing and hangs the window.  Something needs to notify the socket to not reconnect when shutting down.